[2023-01-26 16:48:21,761] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-26 16:48:21,772] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-26 16:48:21,772] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 16:48:21,772] {taskinstance.py:1377} INFO - Starting attempt 7 of 9
[2023-01-26 16:48:21,773] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 16:48:21,788] {taskinstance.py:1397} INFO - Executing <Task(FileSensor): Esperando_el_fichero> on 2023-01-03 00:00:00+00:00
[2023-01-26 16:48:21,794] {standard_task_runner.py:52} INFO - Started process 157 to run task
[2023-01-26 16:48:21,797] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Esperando_el_fichero', 'scheduled__2023-01-03T00:00:00+00:00', '--job-id', '568', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpp4s09lkg', '--error-file', '/tmp/tmpck9a_hzh']
[2023-01-26 16:48:21,799] {standard_task_runner.py:80} INFO - Job 568: Subtask Esperando_el_fichero
[2023-01-26 16:48:21,883] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-03T00:00:00+00:00 [running]> on host 1dac8e18bdf5
[2023-01-26 16:48:21,979] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Esperando_el_fichero
AIRFLOW_CTX_EXECUTION_DATE=2023-01-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=7
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-03T00:00:00+00:00
[2023-01-26 16:48:21,991] {base.py:68} INFO - Using connection ID 'fs_default' for task execution.
[2023-01-26 16:48:21,992] {filesystem.py:58} INFO - Poking for file /tmp/response_20230103.txt
[2023-01-26 16:48:21,992] {filesystem.py:63} INFO - Found File /tmp/response_20230103.txt last modified: 20230126164820
[2023-01-26 16:48:21,992] {base.py:301} INFO - Success criteria met. Exiting.
[2023-01-26 16:48:22,006] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Esperando_el_fichero, execution_date=20230103T000000, start_date=20230126T164821, end_date=20230126T164822
[2023-01-26 16:48:22,049] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 16:48:22,119] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
